include/deal.II/base/function_parser.h

```diff
@@ -26,10 +26,7 @@
 #include <vector>
 #include <map>
 
-namespace mu
-{
-  class Parser;
-}
+#include <muParser.h>
```

Comment:
Let's not agonize too much about this. This is not the first workaround we have for MSVC, and it won't be the last. We know that by about any measure one can apply, it is a pretty bad compiler, but we make the conscious decision to have these workarounds because we want to support Windows builds. So let's just do things as suggested here.